lib/salus/scanners/bundle_audit.rb

```diff
@@ -44,6 +45,18 @@ def run_scanner(scanner, ignore)
         hash = serialize_vuln(result)
         @vulns.push(hash)
 
+        pattern = case hash[:type]
+                  when 'UnpatchedGem'
+                    hash[:name] + ' (' + hash[:version] + ')'
+                  when 'InsecureSource'
+                    hash[:source]
+                  end
+        if !pattern.nil?
```

Comment:
Minor, can you move the line_number logic to a stand alone method

Reply:
done